cumulusci/tasks/salesforce/create_permission_sets.py

```diff
@@ -0,0 +1,35 @@
+from cumulusci.tasks.salesforce.BaseSalesforceApiTask import BaseSalesforceApiTask
+from cumulusci.core.utils import process_list_arg
+
+
+class CreatePermissionSet(BaseSalesforceApiTask):
+    task_options = {
+        "api_name": {
+            "description": "API names of generated Permission Set",
```

Comment:
```suggestion
            "description": "API name of generated Permission Set",
```
Is this suppose to be singular?

Reply:
Yep. Thanks for the catch.